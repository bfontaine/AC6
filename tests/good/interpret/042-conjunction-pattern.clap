val f = case { A[x] and y => { _1 <- x, _2 <- y } } 
val z = f (A[42])

