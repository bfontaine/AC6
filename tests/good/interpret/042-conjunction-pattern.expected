:- f = <code>
:- z = {_1 <- 42, _2 <- A [42] }
